packages/web/src/components/play-bar/desktop/PlayBar.jsx

```diff
@@ -388,9 +389,12 @@ class PlayBar extends Component {
                 isPlaying={isPlaying && !isBuffering}
                 isDisabled={!uid && !collectible}
                 includeTimestamps
+                getAudioPosition={audioPlayer?.getPosition}
+                getTotalTime={audioPlayer?.getDuration}
                 playbackRate={
                   isLongFormContent ? playbackRateValueMap[playbackRate] : 1
                 }
+                // TODO: This can/should also be pulled from the audio player
```

Comment:
This TODO is confusing me a bit, is it not already pulling from the audio player?

Reply:
Oops. I meant to take this out.
But for the sake of explanation: Instead of pulling the values from the audio player and passing them as props (which will re-render the entire child component), the child component can/should just pull the values from the audio player directly and update the relevant elements in the same rAF callback we use for updating the scrubber.

Reply:
Ah I see what you mean